proxy/src/telemetry/metrics/mod.rs

```diff
@@ -49,26 +49,27 @@ use telemetry::event::Event;
 
 mod counter;
 mod gauge;
+mod histogram;
 mod labels;
 mod latency;
 
 use self::counter::Counter;
 use self::gauge::Gauge;
+use self::histogram::Histogram;
 use self::labels::{
     RequestLabels,
     ResponseLabels,
     TransportLabels,
     TransportCloseLabels
 };
-use self::latency::{BUCKET_BOUNDS, Histogram};
 pub use self::labels::DstLabels;
 
 #[derive(Debug, Clone)]
 struct Metrics {
     request_total: Metric<Counter, Arc<RequestLabels>>,
 
     response_total: Metric<Counter, Arc<ResponseLabels>>,
-    response_latency: Metric<Histogram, Arc<ResponseLabels>>,
+    response_latency: Metric<Histogram<latency::Ms>, Arc<ResponseLabels>>,
```

Comment:
NIT/TIOLI: these types are starting to get very long, consider aliasing something like

```rust
type LatencyHist<U, L> = Metric<Histogram<U>, Arc<L>>;

// ...

response_latency: LatencyHist<latency::Ms, ResponseLabels>,
```
or something like that?

Reply:
fair!